.github/workflows/release_nightly.yml

```diff
@@ -164,100 +216,61 @@ jobs:
           path: |
             ${{ env.CARGO_BUILD_DIR }}/ruffle_desktop
             ${{ env.CARGO_BUILD_DIR }}/ruffle_web_safari
-            package
+            common_package
 
   build-mac-universal-binary:
     name: Build macOS universal binary
     needs: [create-nightly-release, build, build-web]
     runs-on: macos-14
     env:
-      PACKAGE_FILE: ${{ needs.create-nightly-release.outputs.package_prefix }}-macos-universal.tar.gz
+      PACKAGE_FILE: ${{ needs.create-nightly-release.outputs.package_prefix }}_macOS_Universal.dmg
+      INTEL_BINARY_PATH: target/x86_64-apple-darwin/release
+      ARM_BINARY_PATH: target/aarch64-apple-darwin/release
+      MIN_MACOS_VERSION: 10.12
     steps:
       - name: Clone Ruffle repo
         uses: actions/checkout@v4
 
-      - name: Download aarch64 binary
+      - name: Download arm64 binary
         uses: actions/download-artifact@v4
         with:
-          name: macos-aarch64
+          name: macOS_arm64
```

Comment:
`arm64` is a non-standard name of `aarch64` (unless it's standard on macos?)

Reply:
[Apple originally named their backend arm64 before they merged.](https://stackoverflow.com/questions/31851611/differences-between-arm64-and-aarch64) But the main reason I changed this is because this name is more commonly known and makes it more intuitive to read and understand the workflow.